Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM l4zy/archlinux:latest
```

Comment:
Why is this not the main archlinux image, but rather a fork? We should not use anything unofficial. Strong -1 here.

Reply:
this is a private image, we can't even audit. I can't support this

Reply:
The main archlinux image did not have systemd on it, this image simply contains systemd on top of main archlinux image
https://hub.docker.com/repository/registry-1.docker.io/l4zy/archlinux/builds/21b85dbe-c21a-4454-85de-284d3c9a5108
![image](https://user-images.githubusercontent.com/21274221/115961385-c57ab300-a533-11eb-9c59-06dbec984607.png)
This was done because, as of now archinstall does not run on systemd less machines #351

Reply:
Still a strong -1. Using ANYTHING not officially shipped by the Arch Linux project is a blocker IMO. AUR is also out of the question.

Reply:
agreed, -1. we can't support this

Reply:
We would need to figure out a way around this. Could we submit an official patch upstream? Is there a reason why they're not running EFI?